script/remove_record.py

```diff
@@ -0,0 +1,19 @@
+import json
+
+
+JSON_FILE = 'init/persons-test.json'
+persons = json.load(open(JSON_FILE, encoding='utf-8'))
+
+
+def remove_record_from_json():
```

Comment:
* Add main function
* Function should do 1 thing only
* do not use global variables